comment/delivery/http/handler.go

```
@@ -0,0 +1,120 @@
+package http
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/PECHIVKO/task-manager/comment"
+	"github.com/go-chi/chi"
+)
+
+type Handler struct {
+	useCase comment.UseCase
+}
+
+func NewHandler(useCase comment.UseCase) *Handler {
+	return &Handler{
+		useCase: useCase,
+	}
+}
+
+type createInput struct {
+	Comment string `json:"comment"`
+	TaskID  int    `json:"task_id"`
+}
+
+func (h *Handler) Create(w http.ResponseWriter, r *http.Request) {
+	input := new(createInput)
+	err := json.NewDecoder(r.Body).Decode(&input)
+	if err != nil {
+		log.Println(err)
+		respondWithError(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+	err = h.useCase.CreateComment(r.Context(), input.Comment, input.TaskID)
+	if err != nil {
+		log.Println(err)
+		respondWithError(w, http.StatusInternalServerError, err.Error())
+	} else {
+		respondwithJSON(w, http.StatusCreated, map[string]string{"message": "Comment successfully created"})
+	}
+}
+
+type updateNameInput struct {
+	CommentID int    `json:"comment_id"`
+	Comment   string `json:"comment"`
+}
+
+func (h *Handler) Update(w http.ResponseWriter, r *http.Request) {
+	input := new(updateNameInput)
+	err := json.NewDecoder(r.Body).Decode(&input)
+	if err != nil {
+		log.Println(err)
+		respondWithError(w, http.StatusInternalServerError, err.Error())
+		return
+	}
+	input.CommentID, _ = strconv.Atoi(chi.URLParam(r, "id"))
```

do not ignore errors